src/registry/inbuilt-components/template-component-map.ts

```diff
@@ -17,4 +18,8 @@ export const templateToComponentMap = [
     name: 'encounter-location',
     baseControlComponent: UiSelectExtended,
   },
+  {
+    name: 'select-concept-answers',
+    baseControlComponent: Dropdown,
```

Comment:
I would use the ui-select-ext

Reply:
@samuelmale  I used the dropdown since it a parity with AFE, but that can change if the `ui-select-ext` is better